### DISCUSSION TITLE
Has 0.8.4 been released or, not?

### DISCUSSION THREAD

**Discussion by 0323pin** [afnanenayet/diffsitter#943] [Q&A] [answered]
Hi,

I see 0.8.4 in crates.io and a tag here on github but, no release here. So, I'd like to know if this should be consider a release or, not.

Thanks!

**Comment by afnanenayet** [comment, marked as answer]
Yeah it should. I forgot to publish the release since the CD pipeline will post it as a draft which I then manually review before clicking publish. I've published it so should be all good now.

**Reply by 0323pin** [reply]
Thanks!
I have a draft update already. Will push it soon :)
